src/game/main.cpp

```diff
@@ -150,24 +154,59 @@ void Check_Windowed(int argc, char *argv[])
         if (strcasecmp(argv[i], "-xpos") == 0) {
             ++i;
             g_xPos = atoi(argv[i]);
-            g_xPos = std::clamp(g_xPos, 0, (int)(Res.right - 800)); // Prevent negative values
         }
 
         if (strcasecmp(argv[i], "-ypos") == 0) {
             ++i;
             g_yPos = atoi(argv[i]);
-            g_yPos = std::clamp(g_yPos, 0, (int)(Res.bottom - 600)); // Prevent negative values
         }
     }
+#ifdef PLATFORM_WINDOWS
+    RECT Res;
+    GetWindowRect(GetDesktopWindow(), &Res);
+    g_xPos = std::clamp(g_xPos, 0, (int)(Res.right - WIN_WIDTH)); // Prevent negative values
+    g_yPos = std::clamp(g_yPos, 0, (int)(Res.bottom - WIN_HEIGHT)); // Prevent negative values
```

Comment:
Now the clamping is Windows specific behaviour, and this code will not run on other platform. This seems not ok.

Reply:
We currently don't use these 2 options for SDL. I just center the window always (SDL_WINDOWPOS_CENTERED). I'll add an else block with a TODO. 

However I'm afraid all of this will require more tinkering for multimonitor setups in SDL2 - which screen to spawn the window on? Which monitor to use for clamping?

That's why i was always centering the window atm

Reply:
Maybe same here, don't edit original function, and just create a new one for SDL2.